### PR TITLE
added wishlist page

### PR DESCRIPTION
**Pull request by Anish-JS**
Added wish list page and also refactored some common CSS

wishlist

### CODE REVIEW

**Review by NtshVrm — APPROVED** [review]
Looks Amazing 👏

File: index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (abhisekgupta001) [inline index.html:7]: Title can be updated.

**Review by abhisekgupta001 — APPROVED** [review]
Looks Good. Deploy it on netlify, that way we can see the deployed preview while reviewing.

**Comment by Anish-JS** [comment]
> Looks Good. Deploy it on netlify, that way we can see the deployed preview while reviewing.

sure will do it.

**Pull request merged**